pkg/monitors/processlist/processlist.go

```diff
@@ -160,9 +161,9 @@ func (m *Monitor) encodeProcess(proc *TopProcess, sampleInterval time.Duration)
 		nice = strconv.Itoa(*proc.Nice)
 	}
 
-	return fmt.Sprintf("\"%d\":[\"%s\",%d,\"%s\",%d,%d,%d,\"%s\",%.2f,%.2f,\"%s\",\"%s\"]",
```

Comment:
Don't you need \\ to send a literal '\'?

Reply:
I replaced the outer quotes for the first are to Sprintf with backticks, which are the raw string delimiter in Go, so you don't need escapes.  The change on this specific line doesn't actually change anything functionally, it just changes the quoting style of the string so it is easier to read.

Reply:
Is there some reason we don't just string.ReplaceAll(..., `"`, `\\"`)?

Reply:
I tried escaping the `"` with both one and two backslashes and our backend still choked on it for some reason (it didn't seem to recognize the escapes).  I confirmed the output to the backend multiple times and it was definitely not liking any double quotes at all for some reason.

Reply:
Can you paste some of the stuff that failed?